libs/akita-ng-entity-service/src/lib/ng-entity-service.spec.ts

```diff
@@ -1,33 +1,34 @@
-import { TestBed, inject, fakeAsync, tick } from '@angular/core/testing';
 import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { fakeAsync, inject, TestBed, tick } from '@angular/core/testing';
+import { NgEntityServiceNotifier } from './ng-entity-service-notifier';
+import { defaultConfig, NgEntityServiceGlobalConfig, NG_ENTITY_SERVICE_CONFIG } from './ng-entity-service.config';
+import { NgEntityServiceLoader } from './ng-entity-service.loader';
 import {
-  TestStore,
-  TestServiceWithDecoratorConfig,
-  TestServiceWithDecoratorAndInlineConfig,
+  storeName,
+  TestEntity,
   TestService,
+  TestServiceWithDecoratorAndInlineConfig,
+  TestServiceWithDecoratorConfig,
   TestServiceWithInlineConfig,
   TestServiceWithMixedConfig,
-  storeName,
-  TestEntity
+  TestStore,
 } from './setup';
-import { NgEntityServiceGlobalConfig, NG_ENTITY_SERVICE_CONFIG, defaultConfig } from './ng-entity-service.config';
-import { HttpMethod, NgEntityServiceNotifier } from './ng-entity-service-notifier';
-import { NgEntityServiceLoader } from './ng-entity-service.loader';
+import { HttpMethod } from './types';
 
 describe('NgEntityService', () => {
   describe('should merge config in order...', () => {
-    describe('', () => {
+    describe(undefined, () => {
```

Comment:
describe(undefined

???

Reply:
having the same description is disallowed https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-identical-title.md

But I was using the describe blocks as a nameless grouping to get the setup hooks to setup correctly for each group.
Using undefined fixes the linting error

Reply:
Disable this rule or give it a name, please. `undefined` is not acceptable.